modules/persistent-storage-csi-drivers-supported.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi.adoc
+
+[id="csi-drivers-supported_{context}"]
+= CSI drivers supported by {product-title}
+
+{product-title} installs certain CSI drivers by default, giving users storage options that are not possible with in-tree volume plug-ins.
```

Comment:
In OCP4.5, aws-ebs and manila csi driver was introduced with 3rd operator, and need user to install manually. Maybe we can have some update here?

Reply:
Oh, yeah...right. I've modified wording here for 4.5 only, PTAL